lab-doug/route/matchscore-router.js

```diff
@@ -0,0 +1,92 @@
+'use strict';
+
+const Router = require('express').Router;
+const debug = require('debug')('matchscore:matchscore-router');
+const matchscoreRouter = module.exports = new Router();
+const AppError = require('../lib/app-error');
+const storage = require('../lib/storage');
+const MatchScoreModel = require('../model/matchscore');
+
+function createMatchscore(reqBody){
```

Comment:
You may want to break things like this out so that you have one file that defines functions like this for your CRUD operations, and one file that handles your routing and calls those functions.